### DISCUSSION TITLE
Slugify gets removed by firewall when running npm init gatsby

### DISCUSSION THREAD

**Discussion by Jukez17** [gatsbyjs/gatsby#33928] [Help]
### Preliminary Checks

- [X] This issue is not a duplicate. Before opening a new issue, please search existing issues: https://github.com/gatsbyjs/gatsby/issues
- [X] This issue is not a question, feature request, RFC, or anything other than a bug report directly related to Gatsby. Please post those things in GitHub Discussions: https://github.com/gatsbyjs/gatsby/discussions

### Description

Not sure if this 100% bug but when doing a fresh project using `npm init gatsby` and choose nothing extras the firewall(f-secure) removes slugify.js file and the `gatsby develop` won't work anymore because the file is missing.

### Reproduction Link

https://github.com/Jukez17/website-testing

### Steps to Reproduce

To reproduce this I believe firewall needs to be active(f-secure) or any other firewall software.

1. run `npm init gasby`
2.  no extra options selected (bare minimum)
3. run `gatsby develop`

### Expected Result

I am expecting to see freshly created bare bone website on my localhost address

### Actual Result

![image](https://user-images.githubusercontent.com/43128825/141117200-227ac852-299f-4290-af64-3a9bddffd134.png)

### Environment

```shell
System:
    OS: Windows 10 10.0.19043
    CPU: (8) x64 Intel(R) Core(TM) i7-1065G7 CPU @ 1.30GHz
  Binaries:
    Node: 16.13.0 - C:\Program Files\nodejs\node.EXE
    Yarn: 1.22.10 - ~\AppData\Roaming\npm\yarn.CMD
    npm: 8.1.3 - ~\AppData\Roaming\npm\npm.CMD
  Languages:
    Python: 3.10.0
  Browsers:
    Edge: Spartan (44.18362.449.0)
  npmPackages:
    gatsby: ^4.1.1 => 4.1.1
  npmGlobalPackages:
    gatsby-cli: 4.1.0
```

**Comment by LekoArts** [comment]
In that case you'll want to exclude the folder from your software then. Nothing really that Gatsby can do about it.

**Reply by Jukez17** [reply]
I did talk with fsecure and send diagnostics just in case but I doubt it would have any trojan. But just in case. If everything is gonna be alright I will do an exception for the thing. Just finding it interesting that I don't know how long this been a thing since I started making the website there wasn't any issues when fsecure was on
